### PR TITLE
Add license-detector data export to the octrc tool.

### PR DESCRIPTION
**Pull request by timemmert**
Add a call to the license detector to octrc. The license detector scans the repository for its used licenses and outputs them in a JSON file.

### CODE REVIEW

**Review by maxhbr — APPROVED** [review]
Looks good. Let me test and probably merge that

**Review by maxhbr — CHANGES_REQUESTED** [review]
Some comments and please add test cases to https://github.com/maxhbr/octrc/blob/main/test:octrc.sh

File: test:octrc.sh [hunk 1]
```diff
@@ -14,7 +14,8 @@ setup() {
 }
 
 teardown() {
-    rm -rf /tmp/octrc-test
+    #rm -rf /tmp/octrc-test
+    echo "Test"
```
Comment (maxhbr) [inline test:octrc.sh:18]: should that be committed?

**Review by maxhbr — CHANGES_REQUESTED** [review]

File: test:octrc.sh (hunk 1 above)
Comment (timemmert) [reply test:octrc.sh:18]: Nope, I will remove that...

**Pull request merged**